pkgs/top-level/python-packages.nix

```
@@ -5153,6 +5153,13 @@ in {
 
   transitions = callPackage ../development/python-modules/transitions { };
 
+  essentia = (toPythonModule (pkgs.essentia.override {
+    pythonPackages = self;
+    pythonSupport = true;
+    # Use our gaia
+    gaia = self.gaia;
```

this could be combined with the other suggestion
```suggestion
    inherit python gaia;
```

---

From some reason I get this error when inheriting `gaia`:

```
error: undefined variable 'gaia' at /var/src/nixpkgs/pkgs/top-level/python-packages.nix:5156:55
```

Where that line is where `essentia` is defined.

---

interesting, well, callPackage should be able to pick up gaia from the current scope. So It might not be needed at all

---

Makes sense, I've pushed a fix but for `python3Packages.essentia` we depend on https://github.com/NixOS/nixpkgs/pull/69310 .